articles/search/hybrid-search-how-to-query.md

```diff
@@ -203,7 +203,7 @@ api-key: {{admin-api-key}}
 
 **Key points:**
 
-+ Semantic ranking accepts up to 50 results from the merged response. Set "k" and "top" to 50 for equal representation of both queries.
```

Comment:
@danquirk for removing the instruction for k/top = 50 for semantic ranking.

Reply:
I removed this to avoid duplication with the section below, which has more details.

In new preview API, to control representation of keyword vs vector in hybrid, you can use:
- keyword results with the maxTextRecallSize
- vector results with k

In the previous APIs, for hybrid:
- keyword results can not be controlled. Please correct me if I am wrong, I can show you my experiments on it but would love to be corrected :)
- vector results are controlled from k

Therefore, I believe the previous wording was incorrect for previous APIs and for the new preview API.